README.rst

```diff
@@ -103,6 +113,33 @@ Using the Python Driver 1.7 and connecting to Neo4j 4.x with default connection
     driver = GraphDatabase.driver("neo4j://localhost:7687", auth=("neo4j", "password"), encrypted=False)
 
 
+Version Compatibility
+=====================
+
++------------------+-------+-------+-------+-------+-------+-------+-------+
+| Server \\ Driver |  1.7  |  4.0  |  4.1  |  4.2  |  4.3  | *4.4* | *5.0* |
++==================+=======+=======+=======+=======+=======+=======+=======+
+| Neo4j 3.5 (EOL)  |   ✔️  |   ✔️  |   ❔   |   ❔   |   ❔   |   ❔   |   ❔   |
++------------------+-------+-------+-------+-------+-------+-------+-------+
+| Neo4j 4.0 (EOL)  |   ✔️  |   ✔️  |   ✔️  |   ✔️  |   ✔️  |   ✔️  |   ❔   |
++------------------+-------+-------+-------+-------+-------+-------+-------+
+| Neo4j 4.1 (EOL)  |   ❔   |   ✔️  |   ✔️  |   ✔️  |   ✔️  |   ✔️  |   ❔   |
++------------------+-------+-------+-------+-------+-------+-------+-------+
+| Neo4j 4.2 (EOL)  |   ❔   |   ❔   |   ✔️  |   ✔️  |   ✔️  |   ✔️  |   ❔   |
++------------------+-------+-------+-------+-------+-------+-------+-------+
+| Neo4j 4.3        |   ❔   |   ❔   |   ❔   |   ✔️  |   ✔️  |   ✔️  |   ❔   |
++------------------+-------+-------+-------+-------+-------+-------+-------+
+| Neo4j 4.4        |   ❔   |   ❔   |   ❔   |   ❔   |   ✔️  |   ✔️  |   ✔️  |
++------------------+-------+-------+-------+-------+-------+-------+-------+
+| Neo4j 5.0        |   ❔   |   ❔   |   ❔   |   ❔   |   ❔   |   ✔️  |   ✔️  |
++------------------+-------+-------+-------+-------+-------+-------+-------+
+
```

Comment:
nitpick: Table not well aligned.

Reply:
It is depending on if your mono font renders  `✔️` and `❔` as mono space characters. I think if it wasn't aligned, it wouldn't even be valid RST. Previously, I tried to work with `Y` and `N` instead because pretty much every pc should have a mono font that supports those, but it made it really difficult to see quickly what is and isn't supported.

Reply:
Found another way. Using `Yes` (supported), `` (blank) (not working), and `?` (maybe working), gives enough visual distinctiveness imo.